diktat-rules/src/test/resources/test/paragraph2/kdoc/ConstructorCommentNewlineExpected.kt

```diff
@@ -0,0 +1,19 @@
+package test.paragraph2.kdoc
+
+/**
+ * @property param1
+ * @property param2  first comment
+*/
+class Example(
+    val param1: String,
+    val param2: String, // second comment
```

Comment:
Initially, I thought, that the main problem is that we mixed up the comments, since they are inline

May be  we can at least open an separate issue for this?

Reply:
Yes, created #1125. For now we are focusing on bugfixes only, and major logic enhancements should be postponed till after 1.0.0.